packages/block-directory/src/store/test/selectors.js

```diff
@@ -23,6 +24,40 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'isRequestingDownloadableBlocks', () => {
+		it( 'should return false if there are no calls pending', () => {
+			const filterValue = 'Awesome Block';
+
+			const state = {
+				downloadableBlocks: {},
+			};
+			const isRequesting = isRequestingDownloadableBlocks(
+				state,
+				filterValue
+			);
+
+			expect( isRequesting ).toEqual( false );
+		} );
+
+		it( 'should return true if at least one call is pending', () => {
+			const filterValue = 'Awesome Block';
+
+			const state = {
+				downloadableBlocks: {
+					[ filterValue ]: {
+						isRequesting: true,
+					},
+				},
+			};
+			const isRequesting = isRequestingDownloadableBlocks(
+				state,
+				filterValue
+			);
+
+			expect( isRequesting ).toEqual( true );
+		} );
```

Comment:
You could add a test for when a request is completed, to cover all cases (this is a totally optional suggestion).

Reply:
I cleaned up the verbiage for the other tests and covered another case. I'm satisfied with that coverage. Cool?